inspirehep/modules/theme/static/scss/format/detailed-record.scss

```diff
@@ -211,6 +211,26 @@ h1.record-detailed-title, .record-header label, #record-journal,
     text-align: center;
 }
 
+.sorting_desc {
+  background-image: url('../images/sort_desc.png');
+  background-repeat: no-repeat;
+  background-position: center right;
+  padding-right: 20px !important;
+}
+
+.sorting {
+  background-image: url('../images/sort_both.png');
+  background-repeat: no-repeat;
+  background-position: center right;
+  padding-right: 20px !important;
+}
+
+.sorting_asc {
+  background-image: url('../images/sort_asc.png');
+  background-repeat: no-repeat;
+  background-position: center right;
+  padding-right: 20px !important;
+}
```

Comment:
Oh! I see. I was confused, because grepping the repository for `sorting_desc` returned 0 results...